src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 5.4
 ---
 
+ * Add the `LogRetryAsWarningInterface` for exceptions to log retries as warnings instead of errors allowing the same behaviour as RecoverableExceptionInterface while still using the retry strategy
```

Comment:
Better move this to the bottom of the list.

Reply:
done